functions/src/CVE/index.ts

```diff
@@ -1,46 +1,89 @@
 import * as functions from 'firebase-functions';
 import * as express from 'express';
 import * as bodyParser from "body-parser";
+import * as admin from 'firebase-admin';
+import * as Enumerable from 'linq';
 
 const app = express();
 const main = express();
 
 main.use(app);
 main.use(bodyParser.json());
 
-
 export const getCVE = functions.https.onRequest(main);
 
 app.get('/cves', (request, response) => {
+  
     const bulletinID = request.query.bulletinid;
-    if (bulletinID !== null){
-      //TODO: call helper function to query for bulletin data 
-    }
     const SPLID = request.query.splid;
```

Comment:
same casing issue with all these variables. Even if it starts with an abbreviation, make the abbreviation all lowercase, i.e.: splID, splStart, cveID, etc.